packages/avs-bot/src/main.ts

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-console */
+import * as path from 'path';
+import {FileEngine} from '@wireapp/store-engine-fs';
+import {Account} from '@wireapp/core';
+import {APIClient} from '@wireapp/api-client';
+import {ClientType} from '@wireapp/api-client/dist/client';
+import {
+  AUTH_TABLE_NAME,
+  AUTH_COOKIE_KEY,
+  AUTH_ACCESS_TOKEN_KEY,
+  AccessTokenData,
+  Cookie,
+  LoginData,
+} from '@wireapp/api-client/dist/auth';
+import {CRUDEngine} from '@wireapp/store-engine';
+import {PayloadBundleType} from '@wireapp/core/dist/conversation';
+import {Avs} from './avs';
+
+const wrtc = require('wrtc');
+
+declare global {
+  namespace NodeJS {
+    interface Global {
+      getUserMedia: NavigatorGetUserMedia;
+      MediaStream: MediaStream;
+      MediaStreamTrack: MediaStreamTrack;
+      navigator: Navigator;
+      nonstandard: {
+        i420ToRgba: unknown;
+        rgbaToI420: unknown;
+        RTCAudioSink: unknown;
+        RTCAudioSource: unknown;
+        RTCVideoSink: unknown;
+        RTCVideoSource: unknown;
+      };
+      RTCDataChannel: RTCDataChannel;
+      RTCDataChannelEvent: RTCDataChannelEvent;
+      RTCDtlsTransport: RTCDtlsTransport;
+      RTCIceCandidate: RTCIceCandidate;
+      RTCIceTransport: RTCIceTransport;
+      RTCPeerConnection: RTCPeerConnection;
+      RTCPeerConnectionIceEvent: RTCPeerConnectionIceEvent;
+      RTCRtpReceiver: RTCRtpReceiver;
+      RTCRtpSender: RTCRtpSender;
+      RTCRtpTransceiver: RTCRtpTransceiver;
+      RTCSctpTransport: RTCSctpTransport;
+      RTCSessionDescription: RTCSessionDescription;
+    }
+  }
+}
+
+if (typeof RTCPeerConnection === 'undefined') {
```

Comment:
IMHO this is AVS related as well, can it be moved to the avs file as well?

Reply:
Essentially there shouldn't be a `wrtc` import in this file.

Reply:
https://github.com/wireapp/wire-web-packages/pull/3070/commits/16edff26574f9ab4f1e8b56685ea1dabf52dedd7